packages/client/hmi-client/src/components/documents/tera-extractions.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+	<Accordion multiple :active-index="[0, 1, 2]">
+		<AccordionTab v-if="!isEmpty(clonedState.equations)">
+			<template #header>
+				<header>
+					Equation Images
+					<span style="font-size: var(--font-caption)">({{ countEquations }} selected)</span>
+				</header>
+			</template>
+			<tera-asset-block
+				v-for="(equation, i) in clonedState.equations"
+				:key="i"
+				:is-included="equation.includeInProcess"
+				@update:is-included="onUpdateInclude(equation)"
+			>
+				<template #header>
+					<h5>{{ equation.name }}</h5>
+				</template>
+				<Image id="img" :src="equation.asset.metadata?.url" :alt="''" preview />
+			</tera-asset-block>
+		</AccordionTab>
+		<AccordionTab v-if="!isEmpty(clonedState.figures)">
+			<template #header>
+				<header>
+					Figure Images
+					<span style="font-size: var(--font-caption)">({{ countFigures }} selected)</span>
+				</header>
+			</template>
+			<tera-asset-block
+				v-for="(figure, i) in clonedState.figures"
+				:key="i"
+				:is-included="figure.includeInProcess"
```

Comment:
Is there still a need to show the 'Include in process' toggle if we are not viewing this in a workflow drilldown?

Reply:
The designers still wanted this toggle to show up, just so when we drag the asset in the drilldown it will already show which extractions they want to use.  We dont actually have the ability to save this state at the moment so its not entirely functional when looking at the asset page